composer.json

```diff
@@ -7,12 +7,12 @@
     "prefer-stable": true,
     "require": {
         "php": ">=7.2",
-        "twig/twig": "^1.34"
+        "twig/twig": "^1.34 || ^2.14"
```

Comment:
Do we really need only latest "twig/twig": "^2.14". I would say it should be `"twig/twig": "^1.34 || ^2.7"` since version <=2.6 have security issues https://packagist.org/packages/twig/twig.

Reply:
Version 1.34 contains security issues, so lowest supported version is 1.38